src/fivetonine/collage/core.clj

```diff
@@ -209,6 +210,35 @@
       (.dispose))
     base))
 
+(defn- triangle-polygon [pos ^long w]
+  (case pos
+    :lower-left (Polygon. (int-array [0 w 0]) (int-array [w w 0]) 3)
+    :lower-right (Polygon. (int-array [w w 0]) (int-array [w 0 w]) 3)
+    :upper-right (Polygon. (int-array [w 0 w]) (int-array [0 0 w]) 3)
+    :upper-left (Polygon. (int-array [0 0 w]) (int-array [0 w 0]) 3)))
+
+(defn triangle
+  "Crop a triangle from an image, leaving a transparent background. Can optionally
+  be called with an option map that specifies where the triangle is positioned:
+
+  ```clj
+  (triangle image {:position :upper-left})
+  (triangle image {:position :lower-left})
+  (triangle image {:position :lower-right})
+  (triangle image {:position :upper-right})
+  ```"
+  [^BufferedImage image & [{:keys [position]}]]
+  (let [width (.getWidth image)
+        circle (BufferedImage. width width BufferedImage/TYPE_INT_ARGB)]
```

Comment:
Hey! Many thanks for all the PRs! 🤩

A minor thing, `circle` should probably really be `triangle`.